src/components/ui/SuggestsDropdown.jsx

```diff
@@ -1,24 +1,31 @@
+/* global _ */
 import React from 'react';
 import classnames from 'classnames';
 import { object, func, string, arrayOf } from 'prop-types';
 
 import SuggestItem from './SuggestItem';
+import { UserFeedbackYesNo } from './index';
 
-const SuggestsDropdown = ({ className = '', suggestItems, onSelect, highlighted }) => {
+const SuggestsDropdown = ({ className = '', suggestItems, onSelect, highlighted, value }) => {
   return (
-    <ul className={classnames('autocomplete_suggestions', className)}>
-      {suggestItems.map((suggestItem, index) => (
-        <li
-          key={index}
-          onMouseDown={() => {
-            onSelect(suggestItem);
-          }}
-          className={classnames({ selected: highlighted === suggestItem })}
-        >
-          <SuggestItem item={suggestItem} />
-        </li>
-      ))}
-    </ul>
+    <>
+      <ul className={classnames('autocomplete_suggestions', className)}>
+        {suggestItems.map((suggestItem, index) => (
+          <li
+            key={index}
+            onMouseDown={() => {
+              onSelect(suggestItem);
+            }}
+            className={classnames({ selected: highlighted === suggestItem })}
+          >
+            <SuggestItem item={suggestItem} />
+          </li>
+        ))}
+      </ul>
+      {value.length > 0 && suggestItems.length > 0 && !suggestItems[0].errorLabel && (
```

Comment:
It's safer to test only `value` for truthiness instead of its length, otherwise it will crash if provided with `null` or `undefined`.